pipeline-braille-scripts/epub3-to-pef/src/main/resources/css/bana.css

```diff
@@ -0,0 +1,175 @@
+/* ------------------------------------------------------------------------- */
+/*  http://www.daisy.org/pipeline/modules/braille/zedai-to-pef/css/bana.css  */
+/* ------------------------------------------------------------------------- */
```

Comment:
Or maybe the HTML one was copied from ZedAI as well? I'd say we delete the `bana.css` and `ueb.css` files that are copies. They were just meant as examples of style sheets that could be present, I don't want to create the impression that we are actually supporting BANA.